tests/test_config_space_from_yaml.py

```diff
@@ -0,0 +1,362 @@
+from copy import deepcopy
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Any, Dict
+
+import pytest
+import yaml
+
+from embeddings.hyperparameter_search.configspace import BaseConfigSpace
+from embeddings.hyperparameter_search.flair_configspace import FlairModelTrainerConfigSpace
+from embeddings.hyperparameter_search.flair_configspace import (
+    SequenceLabelingConfigSpace as FlairSequenceLabelingConfigSpace,
+)
+from embeddings.hyperparameter_search.lighting_configspace import (
+    LightingTextClassificationConfigSpace,
+)
+
+
+@pytest.fixture(scope="module")
+def output_path() -> "TemporaryDirectory[str]":
+    return TemporaryDirectory()
+
+
+@pytest.fixture()
+def base_config_dict() -> Dict[str, Any]:
+    config = {
```

Comment:
fixtures in the file can be in scope "module" because you perform deep copy

Reply:
they cannot because I modify the objects state in assert_config_with_yaml function and I reuse the objects several time for different tests